FirebaseVertexAI/Sources/GenerateContentResponse.swift

```diff
@@ -172,8 +172,8 @@ public struct FinishReason: DecodableProtoEnum, Hashable, Sendable {
 
   /// The token generation was stopped because the response was flagged for safety reasons.
   ///
-  /// > NOTE: When streaming, the ``CandidateResponse/content`` will be empty if content filters
-  /// > blocked the output.
+  /// > NOTE: When streaming, the ``Candidate/content`` will be empty if content filters blocked the
```

Comment:
Note: I missed this in https://github.com/firebase/firebase-ios-sdk/pull/13897 and noticed the warning when rebuilding the docs for this PR.